### ISSUE TITLE
[Component name] I'm not able to prevent dismiss when I choose a wrong date interval

### ISSUE THREAD

**Issue by munsra** [material-components/material-components-android#2839]
The component cannot prevent auto dismiss when I click on positive button.
I'm not able to override setOnClickListener for the confirm_button.

**Comment by paulfthomas** [comment]
Hi @munsra, can you specify the component name? I assume you're talking about `MaterialDatePicker`?

**Comment by munsra** [comment]
Yes, this one: MaterialDatePicker.Builder<Pair<Long, Long>> builderRange = MaterialDatePicker.Builder.dateRangePicker();

**Comment by drchen** [comment]
Can you explain what's your use scenario? Why do you want to prevent auto dismiss?

**Comment by drchen** [comment]
Thanks for the info. I think that makes sense. : )
